Header header
xsensdata         xsens
kellerdata        keller
altimeterdata[2]  altimeter
float32           temp_cabin
float32           temp_wall
float32           temp_water
